locale/en/en.cfg

```diff
@@ -15,6 +15,7 @@ aar-cluster-mode=Cluster mode
 aar-cluster-mode-player=Cluster mode (player override)
 aar-cluster-radius=Cluster radius
 aar-damage-radius-overrides=Damage radius overrides for cluster targeting
+aar-cluster-iterations=Number of clustering iterations
```

Comment:
That's fair. Not sure how to label it. I think we should make 5 or 6 the "high" setting with 10 being like... "Ultra". On the other hand, especially for late game or heavy enemy games, I'd recommend 5 for best performance, so maybe instead of focusing on the 'level' focus on engagement type? Something like

3 = Sparse or Small Enemy Targets
4 = Medium Enemy Presence
5 = Dense Enemies
6-7 = Especially Dense Enemies
10 = Maximum Complexity

Reply:
As for specific numeric values, you probably have a pretty good idea what would work out well.

As for naming (the favourite  bikeshed of them all), not sure that the enemy density would make much sense there. The setting should kinda signal the player that increased precision would mean less shells consumed, but that it might affect game performance a bit.

On the other hand, this could be even a moot point - I don't really know how complex the algorithm is and if it actually makes any kind of difference in gameplay. How about we hard-code the value to 5/6 for starters and see if people complain about performance? And if they do, maybe (re)introduce the setting for it? Is there any practical reason why you would want to run it with higher values (the "good enough" argument)?